src/prefab/construct-env.ts

```diff
@@ -24,6 +24,10 @@ export default async function(pkgenv: { installations: Installation[] }) {
 
   for (const key in rv) {
     rv[key] = rv[key].replaceAll(new RegExp(`\\$${key}\\b`, 'g'), `\${${key}}`)
+    // don’t end with a trailing `:` since that is sometimes interpreted as CWD and can break things
+    // instead of `foo:${PATH}` we end up with `foo${PATH:+:PATH}` which is not POSIX but works
+    // with all realy shells to avoid the trailing `:`
```

Comment:
`s/realy/really/`

Reply:
lol not even sure what I meant to say. “Really” doesn't make sense

Reply:
hm, actually, it's probably `real`.